t/25-job-async-backend.t

```diff
@@ -0,0 +1,135 @@
+use strict;
+use warnings;
+
+use t::TestWSP qw/test_wsp/;
+use curry;
+use Test::More;
+use Test::Fatal;
+use Test::Mojo;
+use JSON::MaybeUTF8 ':v1';
+use Mojo::IOLoop;
+use IO::Async::Loop::Mojo;
+
+our @PENDING_JOBS;
+
+my $loop = IO::Async::Loop::Mojo->new;
+our $LAST_ID = 999;
+package t::SampleClient {
+    use parent qw(Job::Async::Client);
+
+    sub loop { shift->{loop} //= $loop }
+    sub start { my ($self) = @_; }
+
+    sub submit {
+        my ($self, %args) = @_;
+        push @::PENDING_JOBS, my $job = Job::Async::Job->new(
+            data   => $args{data},
+            id     => ++$LAST_ID,
+            future => $self->loop->new_future,
+        );
+        $job->future
+    }
+};
+
+package t::SampleWorker {
+    use parent qw(Job::Async::Worker);
+    use Future::Utils qw(repeat);
+
+    sub loop { shift->{loop} //= $loop }
+
+    sub start { }
+
+    sub trigger : method{
```

Comment:
Why does this need the `:method` attribute?

Reply:
It shouldn't, thanks - will remove. I tend to sprinkle those around the code at random just to pretend that attributes are sometimes useful... this particular code gains nothing from the addition.

Reply:
Done f6e7ed2